src/Annotator.js

```diff
@@ -358,12 +356,12 @@ class Annotator extends EventEmitter {
 
         // Generate map of page to threads
         Object.keys(threadMap).forEach((threadID) => {
-            const annotations = threadMap[threadID];
+            const annotations = threadMap[threadID].sort((a, b) => a.created - b.created);
```

Comment:
Should this use `Date` objects, as well?

Reply:
yup need to move this back to Dates! Didn't work after I tested it :(